src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.jsx

```diff
@@ -82,6 +86,48 @@ const GroupEventToolbar = createReactClass({
     );
   },
 
+  renderRelatedTransactionButton() {
+    const {organization, event, orgId} = this.props;
+
+    const orgFeatures = new Set(organization.features);
+
+    if (!orgFeatures.has('performance-view')) {
+      return null;
+    }
+
+    const transactionTag = event?.tags?.find(tag => {
+      return tag?.key === 'transaction';
+    });
+
+    if (!transactionTag) {
+      return null;
+    }
+
+    const transactionName = transactionTag?.value;
+
+    if (typeof transactionName !== 'string' || !transactionName) {
+      return null;
+    }
+
+    const to = transactionSummaryRouteWithEventView({
+      orgSlug: orgId,
+      transaction: transactionName,
+      projectID: event.projectID,
+    });
+
+    return (
+      <div key="related-transaction">
+        <RelatedTransactionLink
+          className="btn btn-default btn-sm"
```

Comment:
Could we us a Button component here instead of LESS classes.

Reply:
done in https://github.com/getsentry/sentry/pull/18284/commits/c5c4c40f15fe6d9ecf1672611407b3be0a847851